launcher/src/backend/NodeConnection.js

```diff
@@ -857,13 +860,42 @@ export class NodeConnection {
       }
     }
     try {
+      let before = await this.getTimeStamp()
       await this.runPlaybook("Update Stereum", extraVars)
       await this.runPlaybook("Update Changes", {stereum_role: 'update-changes'})
+      let after = await this.getTimeStamp()
+      return parseInt(after) - parseInt(before)
     } catch (err) {
       log.error("Error occurred running stereum updates:\n", err)
     }
   }
 
+  async getTimeStamp(){
+    try {
+      let val = await this.sshService.exec('date +%s')
```

Comment:
You could just take the time diff off of the local desktop. Would save a round to ssh command.

Reply:
I was a bit in a hurry doing this 😅
Improving it as soon as i'm back home 👍🏼